src/main/org/nlogo/compiler/IdentifierParser.scala

```diff
@@ -104,7 +98,7 @@ private class IdentifierParser(program:Program,
     else if(forgiving)
       new _unknownidentifier
     else
-      exception("Nothing named " + varName + " has been defined",
+      exception(I18N.errors.getN("compiler.LocalsVisitor.notDefined", varName),
```

Comment:
Alas, I'm deleting this conditional branch since `forgiving` is getting tossed...